### ISSUE TITLE
Allow to switch between/use multiple databases

### ISSUE THREAD

**Issue by Thomas-Gelf** [Icinga/icingaweb2-module-director#1498]
## Expected Behavior
I want to have multiple distinct Director databases driven by a single Director instance. Depending on my permissions/restrictions I should be allowed to switch between instances. Job Daemon should be made instance-aware.

## Current Behavior
Only on DB can be active at a single point in time. Switching database immediately affects all running web sessions.

**Comment by Thomas-Gelf** [comment]
![director_db-selector](https://user-images.githubusercontent.com/553008/41127824-676bec30-6aac-11e8-8c9c-30933b587b57.png)

**Comment by Thomas-Gelf** [comment]
Hint: forgot d2cfa20 and 10450b1, some more little changes are coming. 35c2c03 is also related.

**Comment by friesoft** [comment]
I somehow can't get this to work. 
I've setup the icinga-vagrant standalone box, added a second mysql database, granted the mysql "director" user access (all privileges) and added the database as new icingaweb2 resource ("director_team2"). I then used director kickstart to populate the database. 

Until this point everything worked fine and I could now use the second director database just fine.

The problem: the selection box on the director dashboard shows only the current resource, not both possible. I'm using the default icingaadmin user so permissions shouldn't be the problem.

My first guess was that I had to manually add the database to the icingaweb2-module-director config (/etc/icingaweb2/modules/director/config.ini) as thats the point where the code seems to get the available resources. 

But I can't figure out the correct format to use. I've tried:
```
[db]
resource = "icingaweb2-module-director, director_team2"
```
and
```
[db]
resource = "icingaweb2-module-director", "director_team2"
```
and various others but the first one seemed the most plausible as it is the format director uses in the ini files for the permission rules array. 

I get the following error:
> Cannot load resource config "icingaweb2-module-director, director_team2". Resource does not exist

Wrong configuration or bug? Shall I open a separate issue for this?
